intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/idea/highlighting/SemanticHighlightVisitor.xtend

```diff
@@ -73,8 +70,9 @@ class SemanticHighlightVisitor implements HighlightVisitor {
 	}
 	
 	override clone() {
-		cloneProvider.get => [
+		getClass.newInstance => [
```

Comment:
My bad - subtypes are supposed to have a constructor with injectMembers(this), right?